config/database.yml

```diff
@@ -8,10 +8,10 @@ default: &default
 
 local: &local
   <<: *default
-  username: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).user : "" %>
-  password: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).password : "" %>
-  host: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).host : "" %>
-  port: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).port : "" %>
+  username: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).user : nil %>
+  password: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).password : nil %>
+  host: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).host : nil %>
+  port: <%= ENV.key?("DATABASE_SERVER") ? URI(ENV.fetch("DATABASE_SERVER")).port : nil %>
```

Comment:
If DATABASE_SERVER is set to "" as now mentioned in the README, each of these lines will be `nil`.

Maybe it's better with just
```erb
<%= URI(ENV.fetch("DATABASE_SERVER")).user %>
```

As the string is now always present :thinking: